server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -73,8 +73,7 @@ public AddNewApplicantReturnObject addNewClient(
       return new AddNewApplicantReturnObject(
           form.withError(
               FORM_FIELD_NAME_EMAIL_ADDRESS,
-              "Email address already in use. Cannot create applicant if an account already"
-                  + " exists."));
+              preferredLanguage.at(MessageKey.ERROR_EMAIL_IN_USE_CLIENT_CREATE.getKeyName())));
```

Comment:
(question, non-blocking) do we have browser tests for this page but with a non-english language?

Reply:
We don't have tests for the translated validation errors.  I thought about adding one, but remembered a conversation we had in the Eng Sync meeting (see notes from 9/18/23) concerning whether to add screenshots of every applicant-facing page in another language so that we can test that all strings are translated.  We decided not to do that because of the bulk it would add to our already-long test suite.  Our decision was, rather, to add a checkbox on the PR template making sure that all new strings were translated.